scheds/rust/scx_layered/src/main.rs

```diff
@@ -1361,7 +1506,11 @@ impl<'a> Scheduler<'a> {
             .max(4);
 
         let calc_frac = |a, b| {
-            if b != 0.0 { a / b * 100.0 } else { 0.0 }
+            if b != 0.0 {
+                a / b * 100.0
+            } else {
+                0.0
+            }
```

Comment:
This is going to keep bouncing depending on who's the last editor. We should set formatting options for the repo.

Reply:
Sorry, I missed that change - I'll undo it.